dolphinscheduler-standalone-server/src/main/resources/application.yaml

```diff
@@ -106,6 +106,14 @@ security:
         email-attribute: mail
         # action when ldap user is not exist (supported types: CREATE,DENY)
         not-exist-action: CREATE
+      ssl:
+        enable: false
+        # jks file path && password
+        # please store the jks file in dolphinscheduler-standalone-server/src/main/resources/ldapkeystore.jks
```

Comment:
This doesn't make sense. Bear in mind that the users don't have source codes at hands.

Reply:
Deleted

Reply:
Please search globally `src/main/resources/ldapkeystore.jks`

Reply:
Done, PTAL